### DISCUSSION TITLE
Button Inside Table

### DISCUSSION THREAD

**Discussion by herakim11** [Budibase/budibase#4761] [Help] [answered]
Can I create a button inside table row?

**Comment by herakim11** [comment, marked as answer]
SOLVED, thank you 🙏🏽😅

**Reply by NicolasFabre81** [reply]
How did you do it ? 
Because when I add my button, if I click it, it enters the row , as I checked the "link table rows" box.
I'd like to have this option, and to have my button to "check" the rows...

![image](https://user-images.githubusercontent.com/28592069/176679856-5ceaca95-7c86-4e3e-a77d-6e86fd5b7e51.png)

**Comment by mincheol-jeong** [comment]
Can I move the button to the right of the row instead of the left?

**Reply by NicolasFabre81** [reply]
I don't think so...

**Reply by koisabhiemail** [reply]
Hi, is there a way now? to move the buttons to the right?

**Reply by melohagan** [reply]
Hey @koisabhiemail You can do this with the community [Super Table](https://github.com/poirazis/bb-component-SuperTable) component.

It's a [custom plugin](https://docs.budibase.com/docs/custom-plugin) .

The Super Table is a modular design that includes "Super Columns" which are containers in of themselves.

Your tree would look something like:

> Super Table
> > Super Column (ID)
> > Super Column (Marque)
> > Super Column (Modele)
> ...
> > Super Column
> > > Button

**Reply by koisabhiemail** [reply]
Thank you :)
